r/tests/testthat/test-dplyr-summarize.R

```diff
@@ -290,16 +290,12 @@ test_that("Functions that take ... but we only accept a single arg", {
   expect_error(call_binding_agg("max", 1, 2), "Multiple arguments to max()")
 })
 
-test_that("median()", {
+test_that("median()", suppressWarnings({
```

Comment:
I like that we're not repeating `suppressWarnings()` a bunch, but I wonder if it might be better in this case to wrap each `compare_dplyr_binding()` separately so it's a bit close to the code we know will be generating too many warnings for us. I'm on the fence and fine to do either way — totally up to you!